### DISCUSSION TITLE
How to change Installer and App Icon

### DISCUSSION THREAD

**Discussion by discoverlance-com** [NativePHP/laravel#376] [Q&A]
I have updated my icons as recommended in the [development app icons](https://nativephp.com/docs/1/getting-started/development#app-icon) section. But after generating the setup.exe, it's using the default nativephp icon and on linux, there's no icon at all.

This is the icon on windows as you can see it's using the default app icon of Native PHP. And also, after installation, the Icon on my desktop is the Native PHP icon.

![image](https://github.com/user-attachments/assets/90853451-1bc5-443e-94ad-ce320bfd9304)

This is the icon after installation:

![image](https://github.com/user-attachments/assets/75fc184e-8851-4a14-9e53-75aee4317aeb)

**Comment by ceaser125** [comment]
hi sir, how can you create an installer for the app .

**Comment by simonhamp** [comment]
This is being tracked by [this issue](https://github.com/NativePHP/laravel/issues/388)
